docker/dockerfile-install-ubuntu

```diff
@@ -0,0 +1,43 @@
+# Parameters related to building hip

```

Comment:
I thought we weren't installing anything, why do we need this file?

Reply:
Good catch. Yeah, I must've added this because it's defined in shared libraries as the install file, but Tensile doesn't use it. Let me delete the file.